apps/web-giddh/src/app/shared/header/components/account-add-new-details/account-add-new-details.component.html

```diff
@@ -183,320 +169,259 @@
-                                                    activeCompany.countryV2?.alpha2CountryCode
-                                                )
-                                                    ? commonLocaleData?.app_zip_code
-                                                    : commonLocaleData?.app_pin_code
-                                            }}</label>
-                                            <input
-                                                type="number"
-                                                id="pinCodeNum"
-                                                class="form-control"
-                                                formControlName="pincode"
-                                                autocomplete="false"
-                                                name="pinCodeNum"
-                                                [placeholder]="
-                                                    zipCodeSupportedCountryList.includes(
-                                                        activeCompany.countryV2?.alpha2CountryCode
-                                                    )
-                                                        ? commonLocaleData?.app_enter_zip_code
-                                                        : commonLocaleData?.app_enter_pin_code
+        <mat-tab-group
+            #staticTabs
+            mat-stretch-tabs="false"
+            id="asidePaneFormTabs"
+        >
+            <mat-tab
+                [label]="localeData?.tabs?.address"
+                *ngIf="!isHsnSacEnabledAcc"
+                class="gstin-address"
+            >
+                <div class="pd-t15 overflow-hidden" formArrayName="addresses">
+                    <ng-container
+                        *ngFor="
+                            let gst of addAccountForm.get('addresses')['controls']
+                                | slice : 0 : gstDetailsLength;
+                            let i = index;
+                            let f = first;
+                            let l = last
+                        "
+                        [formGroupName]="i"
+                    >
+                        <div class="row">
+                            <div class="col-sm-12">
+                                <div class="add-address-default">
+                                    <h3>{{ commonLocaleData?.app_address }} {{ i + 1 }}</h3>
+                                    <div class="remove-radio-btn">
+                                        <span class="pull-left" *ngIf="!f">
+                                            <button
+                                                mat-stroked-button
+                                                color="primary"
+                                                (click)="removeGstDetailsForm(i)"
+                                                class="grey-color btn-danger text-left"
```

Comment:
```suggestion
                                                class="text-left"
```

design check kr lean iske bad

Reply:
Check this